src/test/e2e/PageObjects/CourtList.page.ts

```diff
@@ -1,29 +1,5 @@
 import { CommonPage } from './Common.page';
```

Comment:
Think this CourtList page is not used anymore. Better to remove the class then just left an empty class

Reply:
This page is used to get a title in other tests